src/components/NewCard/README.md

```diff
@@ -0,0 +1,4 @@
+## NewCard
+An implementation of an NewCard for TPAs
+This Readme was auto-generated and needs to be filled manually
```

Comment:
Will need to add a pretty short description and example on how to use the component.
We should also state that this is the formal and correct `Card` to use and that this will be moved to the current `Card` component, which will be deprecated in the near future.